core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/OrientedRectangle2afp.java

```diff
@@ -57,6 +57,22 @@
 		V extends Vector2D<? super V, ? super P>,
 		B extends Rectangle2afp<?, ?, IE, P, V, B>>
 		extends Parallelogram2afp<ST, IT, IE, P, V, B> {
+    /**+
```

Comment:
I don't think so. I have sovled a similar problem by asking to my Eclipse IDE to re-indent the file. In this way it is able to put tab characters at the right place.